synapse/push/mailer.py

```diff
@@ -274,9 +261,7 @@ async def _fetch_room_state(room_id):
             "reason": reason,
         }
 
-        await self.send_email(
-            email_address, "[%s] %s" % (self.app_name, summary_text), template_vars
-        )
+        await self.send_email(email_address, summary_text, template_vars)
```

Comment:
Looks like this was changed so that in some places we can use `app_name` and some in `server_name`?

I guess plus to make the whole thing configurable.

Reply:
Yes, the `[app]` has been moved to the default values to allow more flexibility in the customisation.

Reply:
Makes sense, was just confused by it so wanted to check! 👍